### PR TITLE
Insta-hunger

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

**Comment by wolfreak99** [comment]
While I find this hilariously funny, this is actually a really genius idea.

**Comment by rwjuk** [comment]
Wouldn't this sort of thing be best kept for a plugin script?

**Comment by janisozaur** [comment]
No no no. It _should_ be part of core logic.

**Comment by rwjuk** [comment]
@janisozaur Oh of course, naturally. My mistake. 😛

**Comment by wolfreak99** [comment]
@rwjuk Honestly, it seems like a good opportunity for the official devs to add some gragging rights. It's not too invasive, and as long as not too many pull requests show up from others trying to join the bandwagon, it shouldn't be that harmful. 

If I was to join in, i'd make "wolfreak" either always angry, or prefer train rides.

**Comment by pizza2004** [comment]
I'd say limit it to the Devs that are actually members and have direct commit access to the code.  Even then I'm not sure whether I like it using the username instead of the actual name.  Just my two cents, as I think this is a good joke even if it doesn't make it in. :D

File: src/openrct2/peep/peep.c [hunk 1]
```diff
@@ -12766,6 +12767,12 @@ money32 set_peep_name(sint32 flags, sint32 state, uint16 sprite_index, uint8* te
         peep->peep_flags |= PEEP_FLAGS_HERE_WE_ARE;
     }
 
+    if (peep_check_easteregg_name(EASTEREGG_PEEP_NAME_JANISOZAUR, peep)) {
+        // insta-hunger
+        peep->hunger = 0x0;
```
Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: For variables like this, we use decimal (almost) throughout the game.

**Pull request closed without merging**